ocs_ci/ocs/defaults.py

```diff
@@ -40,6 +40,6 @@
 NOOBAA_STATUS_CONTENT_COUNT = {
     'CustomResourceDefinition': 5, 'Namespace': 1, 'ServiceAccount': 1,
     'Role': 1, 'RoleBinding': 1, 'ClusterRole': 1, 'ClusterRoleBinding': 1,
-    'Deployment': 1, 'NooBaa': 1, 'StatefulSet': 1, 'Service': 2, 'Secret': 3,
+    'Deployment': 2, 'NooBaa': 1, 'StatefulSet': 2, 'Service': 3, 'Secret': 4,
```

Comment:
#1818 Regarding "why is it designed this way"
The code change here is merely to make it compatible with 4.3